AutoCollection/CorrelationContextManager.ts

```diff
@@ -49,7 +53,12 @@ export class CorrelationContextManager {
         if (!CorrelationContextManager.enabled) {
             return null;
         }
-        return Zone.current.get("context");
+        const context = CorrelationContextManager.session.get('context');
```

Comment:
I wonder if we should choose a more AI-specific name than just `context` for this. I know it's also what we used in Zone, and it didn't seem to cause trouble there, but I'm thinking its possible to have conflicts if we're instrumenting an app that already uses cls.